core/scss/components/date-picker/module.scss

```diff
@@ -8,6 +11,126 @@
 @import '../forms/settings';
 @import 'settings';
 
+// // Settings
+@import '../../settings/z-index.scss';
+@import '../input/settings.scss';
+@import '../checkbox/settings.scss';
+@import '../icon/settings.scss';
+@import '../list-item/settings.scss';
+@import '../overlay/settings.scss';
+@import '../radio/settings.scss';
+@import '../toggle-switch/settings.scss';
+
+// // Tools
+@import '../../tools/mixins/border-radius';
+@import '../../tools/mixins/flex';
+@import '../../tools/mixins/focus';
+@import '../../tools/mixins/grid-framework';
+@import '../../tools/mixins/spacing';
+@import '../../typography/mixins';
+@import '../../components/button/mixins';
+@import '../../components/checkbox/mixins';
+@import '../../components/input/mixins';
+@import '../../components/radio/mixins';
+@import '../../components/toggle-switch/mixins';
+
+// // Typography
+@include brand-font-face-regular;
+@include brand-font-face-bold;
+
+// // Component Helpers
+/** @component button */
```

Comment:
Remove the `/**` comments, since they will not be removed in the final CSS output.

Reply:
got this one too, and removed button styles since those will come in independently as a dependency